test/battle/ai_check_viability.c

```diff
@@ -74,17 +77,17 @@ AI_SINGLE_BATTLE_TEST("AI sees increased base power of Grav Apple")
     u32 movePlayer;
     u16 expectedMove;
 
-    PARAMETRIZE { movePlayer = MOVE_CELEBRATE; expectedMove = MOVE_TROP_KICK; }
+    PARAMETRIZE { movePlayer = MOVE_CELEBRATE; expectedMove = MOVE_DRUM_BEATING; }
```

Comment:
The test isn't suppose to pass with Trop Kick and it only passed before due to a flaw in the AI. Logically, there's no reason why the AI should ever pick Trop Kick over a move of the same type that's a base 10 points higher and has a better secondary effect.

Reply:
Can you add an assumption that Drum Beating and Grav Apple have the same base power?

Reply:
The test passed because Grav Apple was considered a non effect move before `Gravity` since the secondary effect wasn't split from it. 

I would disagree it's a flaw in the AI but that's a different convo.

Reply:
> Can you add an assumption that Drum Beating and Grav Apple have the same base power?

No, that's not how the expansion AI works so the assumption would make no sense. 

The expansion AI picks a move based on number of hits to kill + a value calculated by `AI_CompareDamagingMoves`.

Reply:
Ok. Then it's good as is now, right? if so, please mark this conversation as resolved.